app/src/main/java/org/mozilla/focus/utils/UrlUtils.kt

```diff
@@ -151,4 +151,9 @@ object UrlUtils {
 
         return host.substring(start)
     }
+
+    @JvmStatic
+    fun toDisplayUrl(url: String): String {
```

Comment:
nit: -> `toToolbarDisplayUrl`.

I wonder if this should be packaged with other Toolbar related methods because it's unlikely we'll be using this more generally.

Reply:
I don't think we actually have a toolbar in Fire TV.  At least there don't seem to be any Toolbar related classes.